plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/VariationSelectorAttributeOptions.php

```diff
@@ -143,25 +143,26 @@ protected function render_pills( $attributes, $content, $block ) {
 
 		$pills = '';
 		foreach ( $attribute_terms as $attribute_term ) {
-			$pills .= sprintf(
-				'<div %s>%s</div>',
+			$input = sprintf(
+				'<input type="radio" %s/>',
```

Comment:
Do you consider adding some specific class to the input so it's easier to target, rather than using `:has(input[type="radio"]` in CSS? Not a biggie, I'm just curious. If you want keep it minimalistic, it's fine.